ci/prow/config.yaml

```diff
@@ -683,6 +683,140 @@ presubmits:
         secret:
           secretName: covbot-token
 
+  knative/eventing-sources:
+  - name: pull-knative-eventing-sources-build-tests
+    agent: kubernetes
+    context: pull-knative-eventing-sources-build-tests
+    always_run: true
+    rerun_command: "/test pull-knative-eventing-sources-build-tests"
+    trigger: "(?m)^/test (all|pull-knative-eventing-sources-build-tests),?(\\s+|$)"
+    labels:
+      preset-service-account: "true"
+      preset-bazel-scratch-dir: "true"
+      preset-bazel-remote-cache-enabled: "true"
+    spec:
+      containers:
+      - image: gcr.io/knative-tests/test-infra/prow-tests:latest
+        imagePullPolicy: Always
+        args:
+        - "--scenario=kubernetes_execute_bazel"
+        - "--clean"
+        - "--job=$(JOB_NAME)"
+        - "--repo=github.com/$(REPO_OWNER)/$(REPO_NAME)=$(PULL_REFS)"
+        - "--root=/go/src"
+        - "--service-account=/etc/service-account/service-account.json"
+        - "--upload=gs://knative-prow/pr-logs"
+        - "--" # end bootstrap args, scenario args below
+        - "--" # end kubernetes_execute_bazel flags (consider following flags as text)
+        - "./test/presubmit-tests.sh"
```

Comment:
a) You don't have the test scripts in the repo, all your Prow jobs will fail.
b) Please include the CI and coverage postsubmit jobs.
c) Please include the changes to testgrid config as well.

Reply:
added the cron jobs.

Reply:
added test grid changes. I can not run the makefile though